test/gate/printer/graphml_test.cpp

```diff
@@ -0,0 +1,11 @@
+#include "gate/printer/graphml.h"
+#include "gate/model/gnet_test.h"
+
+#include "gtest/gtest.h"
+
+using namespace eda::printer::graphMl;
+
+TEST(toGraphMlTest,all) {
+  GNet test = *makeRand(5, 5);
+  std::cout << test;
```

Comment:
Let's arrange the test similar to test/rtl/parser/ril/ril_test.cpp (using EXPECT_EQ macro).